addons/binding/org.openhab.binding.km200/ESH-INF/thing/device.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="km200"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+   <bridge-type id="kmdevice">
+		<label>KM200/100/50</label>
+		<description>The KM200 binding is communicating with a Buderus Logamatic web KM200 / KM100 / KM50. It is possible to receive and send parameters like string or float values.</description>
+		
+		 <config-description>
+			<parameter name="ip4Address" type="text" required="true">
+				<label>IP4 address</label>
+				<context>network-address</context>
+				<description>IP4 Address of the KMXXX device</description>
+			</parameter>
+			<parameter name="privateKey" type="text" required="false">
+			   <label>Private key</label>
+			   <description>Option 1:Private en-/decryption key built from MD5Salt, GatewayPassword and PrivatePassword</description>
+			 </parameter>
+			<parameter name="md5Salt" type="text" required="false">
+				<label>MD5 salt</label>
+				<description>Option 2: MD5Salt for the building of the private key</description>
+			</parameter>
+			<parameter name="gatewayPassword" type="text" required="false">
+			   <label>Gateway password</label>
```

Comment:
This file looks like a combination of spaces and tabs, please use tabs only. Reformating with the default eclipse config should do the job.